modules/Story/Story.tsx

```diff
@@ -2,7 +2,7 @@ import type { FunctionComponent } from 'react';
 import type { ExtendedStory } from '@prezly/sdk/dist/types';
 import SlateRenderer from 'components/SlateRenderer';
 import { FormatVersion } from '@prezly/sdk/dist/types/Story';
-import StorySeo from '@/components/seo';
+import { StorySeo } from '@/components/seo';
```

Comment:
The story page doesn't have `title` attribute set. I guess it's just not passed to the component.
Other meta seems to be in place